### PR TITLE
User posts

### PR DESCRIPTION
**Pull request by langphil**
User profiles: created a url structure and filtering for individual user profile pages - linked up the 'profile' link in the nav bar also

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -9,7 +9,8 @@ def new
   end
 
   def create
-    @post = Post.create(post_params)
+    @user = User.find(current_user.id)
+    @user.posts.create(post_params)
```
Comment (dbugsy) [inline app/controllers/posts_controller.rb]: You are redirecting, so you don't need the instance variable here - it is never being called in a template.

File: config/routes.rb [hunk 2]
```diff
@@ -1,15 +1,17 @@
 Rails.application.routes.draw do
 
-  devise_for :users  
+  devise_for :users
   get 'welcome/index'
 
-  root 'welcome#index'     
+  root 'welcome#index'
```
Comment (dbugsy) [inline config/routes.rb]: We seem to have 2 root paths defined in this file?

File: db/migrate/20170823162720_add_users_to_posts.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class AddUsersToPosts < ActiveRecord::Migration[5.1]
+  def change
+
+  end
```
Comment (dbugsy) [inline db/migrate/20170823162720_add_users_to_posts.rb]: this migration doesn't do anything?

**Review by dbugsy — CHANGES_REQUESTED** [review]

**Review by petrakh — APPROVED** [review]
:shipit:

**Review by dbugsy — APPROVED** [review]

**Comment by dbugsy** [comment]
:shipit: :shipit:

**Review by ckiteou — APPROVED** [review]
:shipit:

**Pull request merged**